src/domain/Statuses/Status/Status.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import { Props, Variables } from '../../../common'
+import { Text } from '../../Typographies/Text'
+import { StatusDot, StatusDotVariants } from '../StatusDot'
+import { StyledStatus } from './style'
+
+enum Size {
+  Small = 'small',
+  Large = 'large'
+}
+
+interface IStatusProps {
+  /** Component context */
+  componentContext?: string
+  /** What style variant of status dot to display */
+  variant?: StatusDotVariants
+  /** The margins around the component */
+  margins?: Props.Margin
+  /** label of the status */
+  children: string
```

Comment:
wonder if we'll encounter any weird issues with this. do you think children is better to use than a `label` prop?